zap/src/output/luau/client.rs

```diff
@@ -839,7 +855,11 @@ impl<'src> ClientOutput<'src> {
 
 		self.push_studio();
 
-		self.push(include_str!("client.luau"));
+		self.push_check_server();
+
+		self.push_remotes();
+
+		self.push_line("local time = 0");
```

Comment:
This is... awkward! Let's move this whole block of push functions you added into one function call maybe?